lib/datatrans/json/transaction/authorize.rb

```diff
@@ -39,6 +39,10 @@ def request_body
           "errorUrl": params[:error_url]
         }
       }
+
+      body["card"] = params[:card] if params[:card]
+
+      body
```

Comment:
I've alternatively considered:

```rb
def request_body
  auto_settle = params[:auto_settle].nil? ? true : params[:auto_settle]

  {
    "currency": params[:currency],
    "refno": params[:refno],
    "amount": params[:amount],
    "autoSettle": auto_settle,
    "paymentMethods": params[:payment_methods],
    "redirect": {
      "successUrl": params[:success_url],
      "cancelUrl": params[:cancel_url],
      "errorUrl": params[:error_url]
    },
    "card": params[:card]
  }
end
```

But this would mean adjusting the corresponding test scenario, and, more importantly, maybe introduce unintended effects: Absence of entire `card` parameter (current) vs. empty/nil `card` parameter (if `params[:card]` is not provided, which is the case for most scenarios).